DTS-STN/dts-rmp
# Wip api revamp jw

# Description

Removed Node and Nodemon as runners for dev / start script and replaced them with the proper Nuxt scripts. Some work had to be done to the REST API files in order to achieve this. We also cleaned up the `/server` directory and renamed it to `/api`

## Test Instructions

1. View `package.json` and note how the dev script now only executes `nuxt`
1. run `npm run dev`. The app will compile with `nuxt` and start running on the Nuxt dev server
1. Test all functionality expected up to this point in the app. Adding engagements, searching engagements, seeing a list of engagements

## Help Requested

- Look over the way the REST API code is organized. We now use the 'Router' express object instead of an entire 'express' instance for each router. There are a few minor changes in this code. Any suggestions are greatly appreciated.
- Look over the new folder structure for /api (replaces /server). Any suggestions are greatly appreciated.
## Checklist
